From 7e40eebcf43f51cdd2e160b0460548ebc715a4d8 Mon Sep 17 00:00:00 2001
From: Adam Ford <aford173@gmail.com>
Date: Mon, 20 Apr 2020 10:25:47 -0500
Subject: [PATCH 21/21] arm64: dts: beacon-imx8mm-kit: Remove hard-coded serial
 info

The bootloader is capable of passing the baud rate to the kernel
as a boot argument, and the kernel can use this information
to determine the desired baud rate.

This patch removes the hard-coded 115,200 value and the early
serial debug portions from the device tree.

Fixes: PHOENIX8-84

Signed-off-by: Adam Ford <aford173@gmail.com>
---
 arch/arm64/boot/dts/freescale/beacon-imx8mm-kit.dts | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/beacon-imx8mm-kit.dts b/arch/arm64/boot/dts/freescale/beacon-imx8mm-kit.dts
index 8b139491821c..f6d91309ba30 100644
--- a/arch/arm64/boot/dts/freescale/beacon-imx8mm-kit.dts
+++ b/arch/arm64/boot/dts/freescale/beacon-imx8mm-kit.dts
@@ -15,7 +15,6 @@
 	compatible = "fsl,imx8mm";
 
 	chosen {
-		bootargs = "console=ttymxc1,115200 earlycon=ec_imx6q,0x30890000,115200";
 		stdout-path = &uart2;
 	};
 };
-- 
2.17.1

